tubular/scripts/frontend_utils.py

```diff
@@ -102,6 +120,14 @@ def get_npm_aliases_config(self):
             self.LOG('No npm package aliases defined in config.')
         return npm_aliases_config
 
+    def get_npm_private_config(self):
```

Comment:
this can be achieved with `NPM_ALIASES`

Reply:
FWIW, the difference between `NPM_ALIASES` and `NPM_PRIVATE` is that dependencies in `NPM_PRIVATE` are not aliases. If we relied on `NPM_ALIASES`, the syntax might be somewhat unnecessarily verbose/confusing to define an NPM alias for the same package name it already has.

```shell
NPM_ALIASES:
  - "@datadog/browser-rum": "npm:@datadog/browser-rum@2"
```

versus:

```shell
NPM_PRIVATE:
  - "@datadog/browser-rum@2"
```